pkg/controller/networkaddonsconfig/networkaddonsconfig_controller_test.go

```diff
@@ -0,0 +1,166 @@
+package networkaddonsconfig
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/names"
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/render"
+)
+
+type checkUnit struct {
+	key           string
+	shouldExist   bool
+	expectedValue string
+}
+
+var _ = Describe("Networkaddonsconfig", func() {
+	const testDataFile = "testdata/data.yaml"
+	const numOfObjectsInDataFile = 4
+
+	Context("When CR is not labeled", func() {
+		var objs []*unstructured.Unstructured
+
+		BeforeEach(func() {
+			var err error
+			renderData := render.MakeRenderData()
+			objs, err = render.RenderTemplate(testDataFile, &renderData)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(objs)).To(Equal(numOfObjectsInDataFile))
+
+			crLabels := map[string]string{}
+			err = updateObjectsLabels(crLabels, objs)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Should find only default relation labels", func() {
+			appLabelKeys := []checkUnit{
```

Comment:
we should stick to either appLables/relationshipLabels/RelationLabels terminology (not only here, but throughout the PR).
```suggestion
			expectedRelationshipLabels := []checkUnit{
```

Reply:
so i will stick to appLabel which is the one that already done on almost all the files already
its also shorter, and the one we started with

function names do have the word relationship here and there, imo its fine, wdyt ?